wagn/lib/wagn/parser.rb

```diff
@@ -0,0 +1,65 @@
+# -*- encoding : utf-8 -*-
+
+module Wagn
+  class Parser
+    class << self
+      def rspec opts
```

Comment:
Assignment Branch Condition size for rspec is too high. [39.67/15]<br>Method has too many lines. [51/15]